src/plugins/embeddable/public/lib/embeddables/common/constants.ts

```diff
@@ -32,6 +25,14 @@ export const COMMON_EMBEDDABLE_GROUPING: { [key: string]: UiActionsPresentableGr
         defaultMessage: 'Other',
       }),
     getIconType: () => 'empty',
-    order: -1,
+    order: -1, // Given an item that doesn't specify a group is assigned zero, this forces other to come after all intentionally grouped section
+  },
+  legacy: {
```

Comment:
Can you clarify what the difference is between "other" and "legacy"? What is included in "other" and what is included as "legacy"?

Reply:
Sure, "other" is a catch all grouping for any panel action that gets registered without having a group specified for it, more details about the decision to implement this here https://github.com/elastic/kibana/pull/183764/files#r1627891762. Whilst the "legacy" group option contains panels decided on by product to be advertised as such